packages/qwik/src/core/v2/client/vnode.ts

```diff
@@ -931,6 +994,7 @@ export const mapArray_get = <T>(
 
 export const vnode_insertBefore = (
   journal: VNodeJournal,
+  document: Document,
```

Comment:
You should not have to pass `document` around. Please revert this. If you need to get a hold of element you can always say `node.ownerDocument` https://developer.mozilla.org/en-US/docs/Web/API/Node/ownerDocument Because that is a rare case I don't mind reading the DOM for that.